### PR TITLE
Added Noble Integer

### LINKED ISSUE #427: Noble Integer
An integer x is said to be Noble given an array if the number of integers greater than x is equal to x. 
If there are many Noble integers, any of them is returned. If there is no noble integer, then return -1.

Plz assign me this issue @rcoder23

### PR DESCRIPTION
**Pull request by rohanbabbar04**
Fixes #427 

Do review and merge @rcoder23

### CODE REVIEW

File: Algebra/Noble_Integer.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+else:
+    print("Noble Integer =",nobleInteger)
+
+'''
+Sample Output
+Enter the number of elements:4
+Enter the elements:7 3 9 81
+3
+
+'''
```
Comment (rcoder23) [inline Algebra/Noble_Integer.py:32]: recheck output

**Review by rcoder23 — CHANGES_REQUESTED** [review]

**Comment by rcoder23** [comment]
@rohanbabbar04 make issue on data-structure. 
number types question not to much useful

**Comment by rohanbabbar04** [comment]
Ok sir will do

**Comment by rohanbabbar04** [comment]
@rcoder23 Done 
Do review and merge

**Review by rcoder23 — CHANGES_REQUESTED** [review]
in case of input :- 1 2 3 4 5
noble number should  be 1.
but it show 3

**Comment by rcoder23** [comment]
@rohanbabbar04 I search on google but i can't find appropriate explanations about noble number .
So ,I am closing this pull request .
You can work on Segment Tree #433 issue

**Comment by rohanbabbar04** [comment]
Here is the explanation Plz reopen and merge it
https://www.geeksforgeeks.org/noble-integers-in-an-array-count-of-greater-elements-is-equal-to-value/#:~:text=An%20integer%20x%20is%20said,greater%20than%203%20is%20three.

**Comment by rcoder23** [comment]
> in case of input :- 1 2 3 4 5
> noble number should be 1.
> but it show 3

@rohanbabbar04 Solve this

**Comment by rohanbabbar04** [comment]
It is showing No noble integer
In the question, noble integer is an integer where the number of elements greater than n is equal to n
In case of the question integers greater than 1 is equal to so can't be

**Comment by rohanbabbar04** [comment]
> It is showing No noble integer
> In the question, noble integer is an integer where the number of elements greater than n is equal to n
> In case of the question integers greater than 1 is equal to 4 so can't be

@rcoder23 do review

**Pull request merged**